fabric-transfer-api-v1/build.gradle

```diff
@@ -0,0 +1,8 @@
+archivesBaseName = "fabric-transfer-api-v1"
+version = getSubprojectVersion(project, "0.1.0")
```

Comment:
That's actually the case - it's for use before this PR is merged, so that it can be JiJ'ed in mods for testing. I will let modmuss bump the version once it's merged. 😉

Reply:
modmuss needs to be explicitly told about this or it won't bump major

Reply:
Ok, I'm leaving this open and then someone can remind me or modmuss before this is merged.